resources/dicts/patrols/beach/hunting/leaf-fall.json

```diff
@@ -303,8 +303,8 @@
     "chance_of_success": 50,
     "exp": 50,
     "success_text": [
+        "Thinking they could kill a gray fox by themselves, r_c launces into battle! r_c makes it out alive, but so does the gray fox, though upon bringing reinforcements to finish her off, it's discovered she's wisely left the territory with her kits.",
```

Comment:
"launces" needs to be "launches".  Second sentence is also a bit of a run-on sentence, it could be split into two.